libs/sdk-core/src/breez_services.rs

```diff
@@ -1532,7 +1528,12 @@ impl BreezServices {
                          log_message_res = log_stream.message() => {
                           match log_message_res {
                            Ok(Some(l)) => {
-                            info!("node-logs: {}", l.line);
+                            match l.line.to_ascii_lowercase() {
+                                s if s.starts_with("info") => info!("node-logs: {}", l.line.split_at(4).1.trim_start()),
+                                s if s.starts_with("debug") => debug!("node-logs: {}", l.line.split_at(5).1.trim_start()),
+                                s if s.starts_with("unusual") => trace!("node-logs: {}", l.line.split_at(7).1.trim_start()),
```

Comment:
Level mapping of "unusual" level node logs needs to be reviewed, that's why I left this condition even tho it behaves the same with default case.

Here are a couple of sample logs.
`UNUSUAL plugin-gl-plugin-internal: JSON-RPC interface not yet available. Delaying 50ms. IoError(Os { code: 2, kind: NotFound, message: \"No such file or directory\" })`

`UNUSUAL plugin-gl-plugin-internal: Waiting to re-establish a channel: Deadline reached while waiting for at least one channel to be re-established\n\nStack backtrace:\n   0: ...`

`UNUSUAL 0264a62a4307d701c04a46994ce5f5323b1ca28c80c66b73c631dbcb0990d6e835-channeld-chan#4: Adding HTLC 36 too slow: killing connection`

`UNUSUAL 02c811e575be2df47d8b48dab3d3f1c9b0f6e16d0d40b5ed78253308fc2bd7170d-chan#3: Peer permanent failure in CHANNELD_NORMAL: channeld: received ERROR error channel 7a68f688d7b789d29012fcdcf0fa22e966d87d803c322a3ae71c43cdd78c9b09: Funding transaction spent`

`UNUSUAL 02c811e575be2df47d8b48dab3d3f1c9b0f6e16d0d40b5ed78253308fc2bd7170d-chan#1: Peer permanent failure in ONCHAIN: Funding transaction spent`

`UNUSUAL plugin-gl-plugin-internal: Timeout expired waiting`

Should they be upgraded to `Warn` or `Error`?